script/Arcueid/Day 1/01_00_ARC01_2.txt

```diff
@@ -458,7 +456,7 @@ Fighting my way around out there ain't free, I only show my face when I have the
 [sha:0ad0c58e631971a1893811fb558bef35dc420fb7]{
 -- Page 101, Offset 1427.
 -- 派手に出回ってる通り魔事件の話だよ。駅前のセンター街、夜になると警官も巡回しているしな」
-I'm talking about the random murder cases that have been causing a stir recently. There are even cops patrolling around the area in front of the station at night now."
+I'm talking about the street killings that have been causing a stir recently. There are even cops patrolling around the area in front of the station at night now."
```

Comment:
Street killings isn't really vernacular, I'd just make it serial killings/murders

Reply:
maybe "spree killings" for 通り魔事件? The [Wikipedia](https://ja.wikipedia.org/wiki/%E9%80%9A%E3%82%8A%E9%AD%94) version seems to focus mostly on the randomness and I guess spree killer is close to that since it's all about short-time and proximity. "Midnight Spree Killer" also is a media-esque name but doesn't sound as nice as Midnight Street Killer i guess.

alt suggestion: "indiscriminate murders"

Reply:
I changed it because later 深夜の通り魔事件、だっけ comes up as 'The Midnight Street Killer, was it?' and I think they should be the same.

Reply:
That's why I suggested spree killer, but "street killings" doesn't really work.

I don't really think it's of the utmost importance that these be the same since it's really obvious satsuki and arihiko talking about the same thing.

Reply:
Spree it is